frontend/src/routes/Clinic.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import Navbar from '../components/Navbar'
+import ClinicList from '../components/ClinicList'
+import api from '../service/Api'
+
+
+const ClinicView = () => {
+
+    const [data, setData] = React.useState([])
+    React.useEffect(() => { fetch() })
```

Comment:
`fetch` ist eine bereits in JS definierte funktion! Besser wäre hier ein alternativer Name!

Reply:
Gilt für mehrere Stellen im Code!